modules/client/src/test/java/org/apache/ignite/client/fakes/FakeTxManager.java

```diff
@@ -84,7 +84,7 @@ public UUID id() {
             }
 
             @Override
-            public IgniteBiTuple<ClusterNode, Long> enlistedNodeAndTerm(TablePartitionId tablePartitionId) {
```

Comment:
Please remove // TODO: https://issues.apache.org/jira/browse/IGNITE-20377 in PartitionReplicaListener.